sims/verilator/Makefile

```diff
@@ -1,112 +1,175 @@
+#                         this regex will be force enabled in sim output
+#                         default is no selectively enabled printf's
+# VL8_PROFILE           - 'none' if no verilator profiling (default)
+#                         'all' if full verilator runtime profiling
+#                         'threads' if runtime thread profiling only
+# VL8_THREADS           - how many threads the simulator will use (default 1)
+# VL8_FST_MODE          - enable FST waveform instead of VCD
+#============================================================================
+
+
+#============================================================================
 # verilator makefile
-#########################################################################################
+#============================================================================
 ifeq ($(shell which verilator),)
-$(error Did not find Verilator in PATH. Make sure all requirements are installed)
+	$(error Did not find Verilator in PATH. Make sure all requirements are installed)
 endif
 
-#########################################################################################
+#============================================================================
 # general path variables
-#########################################################################################
+#============================================================================
 base_dir=$(abspath ../..)
 sim_dir=$(abspath .)
 
-#########################################################################################
+#============================================================================
 # include shared variables
-#########################################################################################
+#============================================================================
 include $(base_dir)/variables.mk
 
-#########################################################################################
+#============================================================================
 # name of simulator (used to generate *.f arguments file)
-#########################################################################################
+#============================================================================
 sim_name = verilator
 
-#########################################################################################
-# vcs simulator types and rules
-#########################################################################################
+#============================================================================
+# verilator simulator types and rules
+#============================================================================
 sim_prefix = simulator
 sim = $(sim_dir)/$(sim_prefix)-$(MODEL_PACKAGE)-$(CONFIG)
 sim_debug = $(sim_dir)/$(sim_prefix)-$(MODEL_PACKAGE)-$(CONFIG)-debug
 
-PERMISSIVE_ON=
-PERMISSIVE_OFF=
-
-WAVEFORM_FLAG=-v$(sim_out_name).vcd
```

Comment:
This is used in `common.mk` and shouldn't be deleted.

Reply:
yep, fixed